Python/ceval_macros.h

```diff
@@ -234,6 +234,11 @@ GETITEM(PyObject *v, Py_ssize_t i) {
 #define STACK_SHRINK(n)        BASIC_STACKADJ(-(n))
 #endif
 
+
+/* Data access macros */
+#define CO_CONSTS (frame->f_code->co_consts)
+#define CO_NAMES  (frame->f_code->co_names)
```

Comment:
I'd rename the macros to `FRAME_CO_CONSTS` or `FRAME_CONSTS` (etc.), to make it a little more obvious what they do? Also add a comment explaining that these macros are a crutch to make the generator recognize that these are used.